### PR TITLE
TR-1051 Remove the calls to individual container instances / container list

### PR DESCRIPTION
**Pull request by awinder**

### CODE REVIEW

File: ecs-deploy [hunk 1]
```diff
@@ -357,13 +357,30 @@ function updateService() {
             # Scan the list of running tasks for that service, and see if one of them is the
             # new version of the task definition
 
-            RUNNING_TASKS=$($AWS_ECS list-tasks --cluster "$CLUSTER"  --service-name "$SERVICE" --desired-status RUNNING \
+            RUNNING_TASKS=$($AWS_ECS list-tasks --cluster "$CLUSTER" --desired-status RUNNING \
```
Comment (pmcclory) [inline ecs-deploy]: am I missing where we're filtering on the service?

Comment (pmcclory) [reply ecs-deploy]: oh maybe :homerbush: I see it's handled by the `jq` statement below I think

**Comment by awinder** [comment]
Yep just tested from my local terminal with a sparkposttest deploy:

```
Using image name: ---.dkr.ecr.us-west-2.amazonaws.com/transmissions-api:d6cc924f817966ba5992e597db9573167f8a400e
Current task definition: arn:aws:ecs:us-west-2:----:task-definition/transmissions-api-sparkposttest-resources-ECSTask-1LF2BSNDN6YIY:30
New task definition: arn:aws:ecs:us-west-2:------:task-definition/transmissions-api-sparkposttest-resources-ECSTask-1LF2BSNDN6YIY:31
Service updated successfully, new deployment started.
Waiting for service deployment to complete...
Service deployment successful.
```

**Review by pmcclory — APPROVED** [review]
LGTM.  wonder if this is something we should put a PR upstream for?

**Comment by awinder** [comment]
oh yeah good call.  i'll do that, gonna have to re-do it on a new branch though because we've diverged from upstream

**Pull request merged**